third_party/gems/gemfile.bzl

```diff
@@ -0,0 +1,38 @@
+def _parse_package(line):
+    """
+    Parse an exact package specification from a single line of a Gemfile.lock.
+    If the line did not contain an exact specification (something of the form
+    `package (version)`, return None.
+    """
+
+    prefix = line[0:4]
```

Comment:
I would love to get a comment on these lines that looks like what you expect the string to look like when it's being parsed.

Reply:
There is a description in backticks in the comment, but I'll add a more thorough example.

Reply:
I've expanded the comment, please take another read through.